test/integration/cve_int_test.go

```diff
@@ -56,10 +56,6 @@ func (suite *CveIntegrationTestSuite) TestCveNoVulnerabilities() {
 	cp := ts.Spawn("cve")
 	cp.Expect("No CVEs detected")
 	cp.ExpectExitCode(0)
-
-	cp = ts.Spawn("cve", "report") // legacy alias
-	cp.Expect("No CVEs detected")
-	cp.ExpectExitCode(0)
```

Comment:
I'm pretty sure we want to keep asserting that this works. Is there a specific reason you removed it?

Reply:
I'm not sure what our policy is regarding legacy alias testing. I don't know that a CVE test is the appropriate place to test that aliases work.

Reply:
I wouldn't know where else to test something like this. I won't press the point further though.

Reply:
Whether or not the aliasing works should be tested in it's own test, ideally a unit test. 

I did a search and didn't find it there so I will file a follow-up story for us to add them. Part of that story should be a discussion amongst us to determine if we want to enforce alias testing in our integration tests as an added check to prevent removal of these aliases.